targets/targets.json

```diff
@@ -1465,10 +1465,11 @@
             }
         },
         "macros_add": ["RTC_LSI=1", "HSE_VALUE=12000000", "GNSSBAUD=9600"],
-        "device_has_add": ["ANALOGOUT", "TRNG"],
+        "device_has_add": ["ANALOGOUT", "SERIAL_FC", "TRNG", "FLASH"],
```

Comment:
I can put that change in another PR if you prefer but I *really* don't want to wait as we definitely need flow control on the serial ports.